lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -58,13 +47,13 @@ module.exports = {
               albObj.conditions.method = [].concat(event.alb.conditions.method);
             }
             if (event.alb.conditions.header) {
-              albObj.conditions.header = this.validateHeaderCondition(event, functionName);
+              albObj.conditions.header = event.alb.conditions.header;
             }
             if (event.alb.conditions.query) {
-              albObj.conditions.query = this.validateQueryCondition(event, functionName);
+              albObj.conditions.query = event.alb.conditions.query;
             }
             if (event.alb.conditions.ip) {
-              albObj.conditions.ip = this.validateIpCondition(event, functionName);
+              albObj.conditions.ip = event.alb.conditions.ip;
```

Comment:
We need to maintain support for `string` notation, so it should be `[].concat(event.alb.conditions.ip)`